### ISSUE TITLE
Ch 3. Line 77 in Python Code

### ISSUE THREAD

**Issue by deulkkae** [gedeck/practical-statistics-for-data-scientists#23]
https://github.com/gedeck/practical-statistics-for-data-scientists/blob/0db4dbbcdb3ea61f2adb29e3912873168cf0bb92/python/code/Chapter%203%20-%20Statistial%20Experiments%20and%20Significance%20Testing.py#L77

This line brings typeerror: TypeError: '>' not supported between instances of 'list' and 'float'

It would be better to correct this line to
`print(np.mean(np.array(perm_diffs) > mean_b - mean_a))`

**Comment by gedeck** [comment]
This is strange. It works in the notebook:
![image](https://user-images.githubusercontent.com/8720575/118934307-82e9a200-b94a-11eb-92a0-52167b0941af.png)

I looked into this and it works for me because in the notebook, the type of `mean_b` and `mean_a` is `numpy.float64`. In this case, numpy will implicitly convert the list to np.array and make it work. Your error message suggests that the means are `float` on your system. The floats are from:
```
session_times[session_times.Page == 'Page B'].Time.mean()
```

What Python, numpy and pandas versions are you using? I tested now using Python 3.9.4, numpy 1.20.2, and pandas 1.2.4.
